packages/minter-contracts/ligo/src/fixed_price_sale/fixed_price_sale_market.mligo

```diff
@@ -107,12 +107,12 @@ let fixed_price_sale_main (p, storage : market_entry_points * storage) : operati
      let u = fail_if_paused(storage.admin) in
      buy_token(sale, storage)
   | Cancel sale ->
-     let is_seller = Tezos.sender = sale.sale_seller in
+     let is_seller = Tezos.sender = sale.seller in
```

Comment:
`is_seller` is unnecessary. The check already happens in `fail_if_not_admin_ext` here https://github.com/tqtezos/minter-sdk/blob/7d814e5e3b76179c4d82a742373aab25be90c951/packages/minter-contracts/ligo/fa2_modules/admin/simple_admin.mligo#L25-L29